src/com/oltpbenchmark/api/TPCCTableSchemas.java

```diff
@@ -0,0 +1,147 @@
+package com.oltpbenchmark.api;
+
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import com.oltpbenchmark.benchmarks.tpcc.TPCCConstants;
+
+public class TPCCTableSchemas {
+    public static final Map<String, TableSchema> tables = Stream.of(
```

Comment:
please fix indenting of these builders. would be easier to glance at if the `.foo()` calls below the initial line of each builder were indented

Reply:
Done